### PR TITLE
docs: Doc Updates for 1.11

### PR DESCRIPTION
**Pull request by jfermi**
Add Blog Post. Update images in welcome, install, UI, features and form editor pages. Complete additions for Side by Side Editor, CRD, compare all and Git Integration.

This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/git-integration.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Git Integration for DevOps & GitOps
+
+Git is integrated into Monokle to tackle GitOps seamlessly within the application. If you work within a DevOps culture our Git integration provides advantages for you and your team by keeping a history of changes made to your applications stored in Git to ensure access to a reliable audit trail of your changes.
```
Comment (chargio) [inline docs/git-integration.md:3]: Monokle integrates with Git to provide support for Git based flows, including GitOps. If you using DevOps techniques and culter, our Git integratrion provides a history of changes made to your applications, with a full audit trail of your changes.

File: docs/git-integration.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## **Getting Started with a New Project: Clone a Git repository**
+
+You can use a Git repository URL to start a new project. The repository will be cloned to the selected location and then opened into Monokle.
```
Comment (chargio) [inline docs/git-integration.md:7]: You can start a new clone form a Git repository URL. The repo will be cloned locally and then opened into Monokle

File: docs/git-integration.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+![Clone Git Repo](img/clone-a-git-repo-on-startup-1.11.png)
+
+This allows multiple colleagues to debug, validate, and diff all of your configuration files taking full advantage of all of the features of Monokle to achieve your desired state while also maintaining clear control of your version changes.
```
Comment (chargio) [inline docs/git-integration.md:11]: You can use the collaboration support of Git to debug, validate, and diff all your configuration files to define and store your desired state with a full audit trail. All features of Monokle are already or in the process to fully function with Git.

**Review by chargio — CHANGES_REQUESTED** [review]
Some minor changes

File: docs/form-editor.md [hunk 4]
```diff
@@ -4,11 +4,29 @@
...
 
-![Form Editor](img/form-editor-1.9.png)
+![Form Editor](img/form-editor-1.11.png)
 
 For example in the case of a ConfigMap Kubernetes resource, clicking on the **Form** tab at the top of the Editor, opens the form fields editor for key ConfigMap properties, allowing you to edit/discover all available properties without having to learn or lookup the corresponding YAML/resource documentation. Any changes made and saved (with the Save button on the top right)
 are written back to the underlying YAML. 
 
+### **Modified Values Highlighted in Code Editor**
+
+To have insight into the changes made to your configuration, the exact properties edited with Monokle’s Form Editor are highlighted when you make the transition from the Form Editor tab to the Source tab. This highlighting feature identifies all changes providing full visibility of edited resources as you work toward attaining your desired state.
```
Comment (jfermi) [inline docs/form-editor.md:14]:
```suggestion
When using forms, in order to have insights into the actual changes made to your configuration files, the exact properties edited with Monokle’s Form Editor are highlighted when you make the transition from the Form Editor tab to the Source tab. This highlighting feature identifies all changes providing full visibility of edited resources as you work toward attaining your desired state.
```

File: docs/form-editor.md [hunk 5]
```diff
@@ -4,11 +4,29 @@
...
+
+![Form Editor Highlights](img/form-editor-highlights-1.11.png)
+
+### **Side-by-Side Editors**
+
+The configuration file you are editing can be visible even when you are using templates.
```
Comment (jfermi) [inline docs/form-editor.md:20]:
```suggestion
The configuration file you are editing can be visible even when you are using forms.
```

File: docs/git-integration.md (hunk 1 above)
Comment (jfermi) [inline docs/git-integration.md:3]:
```suggestion
Monokle integrates with Git to provide support for Git based flows, including GitOps. If you using DevOps techniques and culture, our Git integration provides a history of changes made to your applications, with a full audit trail of your changes.
```

File: docs/git-integration.md (hunk 3 above)
Comment (jfermi) [inline docs/git-integration.md:11]:
```suggestion
You can use the collaboration support of Git to debug, validate, and diff all your configuration files to define and store your desired state with a full audit trail. All features of Monokle are already or in the process to fully function with Git.
```

Comment (jfermi) [reply docs/git-integration.md:11]: Changed.

File: docs/git-integration.md (hunk 2 above)
Comment (jfermi) [inline docs/git-integration.md:7]:
```suggestion
You can start a new clone from a Git repository URL. The repo will be cloned locally and then opened into Monokle.
```

Comment (jfermi) [reply docs/git-integration.md:7]: Changed.

**Review by jfermi — COMMENTED** [review]
Made suggested changes.

**Review by topliceanurazvan — APPROVED** [review]

**Pull request merged**